README.md

```diff
@@ -261,18 +261,34 @@ strategies:
 
 ### PodLifeTime
 
-This strategy evicts pods that are older than `.strategies.PodLifeTime.params.maxPodLifeTimeSeconds` The policy
-file should look like:
+This strategy evicts pods that are older than `maxPodLifeTimeSeconds`. The policy file should look like:
 
-````
+```
 apiVersion: "descheduler/v1alpha1"
 kind: "DeschedulerPolicy"
 strategies:
   "PodLifeTime":
      enabled: true
      params:
-        maxPodLifeTimeSeconds: 86400
-````
+       podLifeTime:
+         maxPodLifeTimeSeconds: 86400
+```
+
+You can specify `podStatusPhases` to evict pods with specific `StatusPhases`, currently this parameter is limited to
```

Comment:
This could change to be clear that it will evict *only* the pods with these specific statusphases. I would also change the example to `Pending`, since Running is the default and explain this would only apply the strategy to Pending pods